src/schema/v2/home/__tests__/home_page_artist_module.test.js

```diff
@@ -82,15 +96,15 @@ describe("HomePageArtistModule", () => {
     it("returns trending artists", () => {
       return runAuthenticatedQuery(query("TRENDING"), context).then(
         ({ homePage }) => {
-          expect(homePage.artistModule.results).toEqual([{ slug: "trending" }])
+          expect(homePage.artistModule.results[0].slug).toEqual("trending")
         }
       )
     })
 
-    it("returns trending artists", () => {
-      return runAuthenticatedQuery(query("TRENDING"), context).then(
+    it("returns popular artists", () => {
```

Comment:
This, and the other one like it on line 74, were copy and pasted but never modified.